src/main/java/model/Car.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+import utils.RandomNumberGenerator;
+
+public class Car {
+    private String name;
```

Comment:
불변가능한 변수는 `final`로 표현해도 좋겠네요!
```suggestion
    private final String name;
```